### PR TITLE
New package: vpnd-0.1.1

### PR DESCRIPTION
**Pull request by emacsomancer**
package notifier for Void

### CODE REVIEW

File: srcpkgs/vpnd/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="Benjamin Slade <slade@lambda-y.net>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/emacsomancer/vpnd"
+distfiles="https://gitlab.com/emacsomancer/vpnd/-/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=49e148766f1e0bdb4a48d58bd72448e72c977f4c326393d694ae0cce2b42f6f9
+
+do_install() {
+	vbin vpnd
+	vmkdir usr/share/icons/vpnd
+	vcopy *.png usr/share/icons/vpnd
```
Comment (Piraty) [inline srcpkgs/vpnd/template:17]: quote the glob

File: srcpkgs/vpnd/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	vmkdir etc
+	vcopy vpnd.conf etc
```
Comment (Piraty) [inline srcpkgs/vpnd/template:19]: there is vconf

File: srcpkgs/vpnd/template (hunk 1 above)
Comment (emacsomancer) [reply srcpkgs/vpnd/template]: fixed in https://github.com/void-linux/void-packages/commit/4a686b34bf2eeaf74359eaa334a040f7b529d1dd

File: srcpkgs/vpnd/template (hunk 2 above)
Comment (emacsomancer) [reply srcpkgs/vpnd/template]: fixed in https://github.com/void-linux/void-packages/commit/4a686b34bf2eeaf74359eaa334a040f7b529d1dd

**Pull request merged**